recipes/osx-sysroot/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "macosx-sysroot" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+build:
+  noarch: generic
+  number: 0
+
+outputs:
+  - name: sysroot_osx-64
+    version: {{ sdk_version }}
+    run_exports:
+      - __osx >={{ sdk_version }}
+    test:
+      commands:
+        - echo "hi!"
+
+  - name: sysroot_osx-arm64
+    version: {{ arm_sdk_version }}
+    run_exports:
+      - __osx >={{ arm_sdk_version }}
+    test:
+      commands:
+        - echo "hi!"
```

Comment:
Since `sysroot_{{ cross_target_platform }}` would handle this case, we can drop it

```suggestion
```